src/SFA.DAS.ApprenticeCommitments/Data/Models/Apprenticeship.cs

```diff
@@ -30,6 +30,7 @@ private Apprenticeship()
         public bool? TrainingProviderCorrect { get; private set; }
         public bool? EmployerCorrect { get; private set; }
         public bool? ApprenticeshipDetailsCorrect { get; private set; }
+        public bool? HowApprenticeshipDeliveredCorrect { get; private set; }
```

Comment:
Does this already exist in the database project?  I think it may need to be added to `Apprenticeship.sql`

Reply:
Thanks for this, I've added it in